databases/week2/homework.sql

```diff
@@ -0,0 +1,134 @@
+
+-- Part 1
+-- Add a task with these attributes: title, description, created, updated, due_date, 
+-- status_id, user_id
+
+insert into task (title, description, created, updated, due_date, status_id, user_id)
+values ("more", "more investments", "2017-10-15 13:05:09", "2017-10-15 13:05:09","2017-10-15 13:05:09", 3, 1)
+
+-- Change the title of a task
+UPDATE task SET title = "investements" WHERE id = 1;
+-- Change a task due date
+UPDATE task SET due_date = "2017-10-15 13:05:09" WHERE id = 1;
+-- Change a task status
+SELECT title FROM task WHERE id = 1;
+UPDATE task SET status_id = 1 WHERE id = 1;
+-- Mark a task as complete
+UPDATE task SET status_id = 3, description = "completed" WHERE id = 1;
```

Comment:
I think this was more meant as change status to 'done'. Don't hardcode the status_id, but try to use a query for it.